pom.xml

```diff
@@ -112,11 +112,18 @@
 
         <!-- Spark versions -->
         <spark_2.version>2.4.3</spark_2.version>
-        <spark_3.version>3.1.2</spark_3.version>
+        <spark_3.version>3.2.1</spark_3.version>
```

Comment:
We should upgrade it to 3.2.2 because of https://nvd.nist.gov/vuln/detail/CVE-2022-33891

Reply:
Done.